lib/dl_formula/dl_formula/definitions/functions_array.py

```diff
@@ -556,6 +556,18 @@ class FuncArrayRemoveDefault(FuncArrayRemoveBase):
     ]
 
 
+class FuncArrayIntersect(ArrayFunction):
+    name = "arr_intersect"
+    arg_names = ["array_1", "array_2", "array_3"]
+    arg_cnt = None
+    argument_types = [
+        ArgTypeForAll(DataType.ARRAY_STR, require_type_match={DataType.ARRAY_STR, DataType.CONST_ARRAY_STR}),
+        ArgTypeForAll(DataType.ARRAY_INT, require_type_match={DataType.ARRAY_INT, DataType.CONST_ARRAY_INT}),
+        ArgTypeForAll(DataType.ARRAY_FLOAT, require_type_match={DataType.ARRAY_FLOAT, DataType.CONST_ARRAY_FLOAT}),
+    ]
+    return_type = FromArgs(0)
```

Comment:
Let's hide this new function from docs & suggest until we test it and prepare docs, see https://github.com/datalens-tech/datalens-backend/pull/706/files#diff-f4aab05630b38419fce21c88707365629fef656c75975553727dbb80eb22f9bbR281

Reply:
Everything else looks fine